src/postprocess/band_structure.jl

```diff
@@ -19,18 +19,20 @@ function high_symmetry_kpath(model; kline_density=20)
     # https://github.com/louisponet/DFControl.jl/blob/master/src/structure.jl
     # but for this the best way to go would be to refactor into a small "CrystalStructure"
     # julia module which deals with these sort of low-level details everyone can agree on.
-    pystructure = pymatgen_structure(model.lattice, model.atoms)
-    symm_kpath = pyimport("pymatgen.symmetry.bandstructure").HighSymmKpath(pystructure)
-    kcoords, labels = symm_kpath.get_kpoints(kline_density, coords_are_cartesian=false)
+    std_lattice = spglib_get_std_lattice(model.lattice, model.atoms)
+    sgnum = spglib_get_spacegroup(model.lattice, model.atoms)
+    Rs = [std_lattice[i, :] for i in 1:size(std_lattice,1)]
+    kp = Brillouin.irrfbz_path(sgnum, Rs)
+    kcoords = Brillouin.interpolate(kp, density=kline_density)
```

Comment:
I think there is still a unit problem here (in lattice versus kline_denisty) and also there is the edge with the respective lattices to figure out (conventional versus primitive ...). I'll do some research on this.